### PR TITLE
ceph_disk/main.py: FreeBSD root has wheel for group

### PR DESCRIPTION
**Pull request by wjwithagen**
Signed-off-by: Willem Jan Withagen <wjw@digiware.nl>

### CODE REVIEW

File: src/ceph-disk/ceph_disk/main.py [hunk 1]
```diff
@@ -4857,10 +4859,11 @@ def main_trigger(args):
...
-        ('/usr/bin/ceph-mon', 'root', 'root', True, False),
-        ('/usr/bin/ceph-mds', 'root', 'root', True, False),
-        ('/usr/bin/ceph-osd', 'root', 'root', True, False),
-        ('/usr/bin/radosgw', 'root', 'root', True, False),
+        ('/usr/bin/ceph-mon', 'root', ROOTGROUP, True, False),
+        ('/usr/bin/ceph-mds', 'root', ROOTGROUP, True, False),
+        ('/usr/bin/ceph-osd', 'root', ROOTGROUP, True, False),
+        ('/usr/bin/radosgw', 'root', ROOTGROUP, True, False),
+        ('/etc/ceph', 'root', ROOTGROUP, True, True),
         ('/etc/ceph', 'root', 'root', True, True),
```
Comment (liewegas) [inline src/ceph-disk/ceph_disk/main.py]: remove old /etc/ceph line?

Comment (wjwithagen) [inline src/ceph-disk/ceph_disk/main.py]: @liewegas 
Fixed and repushed.

**Comment by liewegas** [comment]
retest this please

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2017-07-27_15:41:31-ceph-disk-wip-kefu-testing2-distro-basic-vps/

**Comment by tchaikov** [comment]
retest this please.

**Pull request merged**